colibre/scripts/stellar_abundances.py

```diff
@@ -274,9 +311,13 @@ def read_data(data, xvar, yvar):
         ymax = 2
     else:
         raise AttributeError(f"No APOGEE dataset for x variable {xvar}!")
-    obs_data = load_observations([observational_data])[0]
-    x = obs_data.x
-    y = obs_data.y
+
+    # Reading APOGEE data
+    obs_data = h5py.File(observational_data, "r")
```

Comment:
Should this file be closed later in the script?

Reply:
Yes! done